### PR TITLE
QField OAuth2 support

### PR DESCRIPTION
**Pull request by nirvn**
Let the CI tells us if it compiles with QCA and QtKeyChain, and if so let APKs rain.

### CODE REVIEW

**Comment by nirvn** [comment]
(It requires an OSGeo4A SDK update, coming later today)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for df9845fa83db7007566a05c96c8852672cd71af3:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8ec396b77e0729ec1091b56ee253ad6aa0760d77-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8ec396b77e0729ec1091b56ee253ad6aa0760d77-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8ec396b77e0729ec1091b56ee253ad6aa0760d77-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8ec396b77e0729ec1091b56ee253ad6aa0760d77-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1792d9da4be8fb832914efb1c8d69d6d7cc37d58:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-67423987b38667ab8a8f155b8f4b424178fe2a33-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-67423987b38667ab8a8f155b8f4b424178fe2a33-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-67423987b38667ab8a8f155b8f4b424178fe2a33-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-67423987b38667ab8a8f155b8f4b424178fe2a33-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 27893d3dab29ba08541ad147e46d863ea54a716e:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4f47c296604fd044356a466dbda011af622146f1-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4f47c296604fd044356a466dbda011af622146f1-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4f47c296604fd044356a466dbda011af622146f1-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4f47c296604fd044356a466dbda011af622146f1-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0a3b0d55cb85c2b66a28a6b99aaa3730cb380d70:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-7d333e11b66227ac33cceda08bac74bcd142566a-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-7d333e11b66227ac33cceda08bac74bcd142566a-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-7d333e11b66227ac33cceda08bac74bcd142566a-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-7d333e11b66227ac33cceda08bac74bcd142566a-x86.apk)

**Comment by nirvn** [comment]
Last piece of the puzzle: https://github.com/qgis/QGIS/pull/43262

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for eb5fdae18edf30b68f0104e368fd2d5ff59c7c0c:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19262a06ef11313979e787291c35f297e2368745-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19262a06ef11313979e787291c35f297e2368745-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19262a06ef11313979e787291c35f297e2368745-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19262a06ef11313979e787291c35f297e2368745-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for ab89b88a9fdb7daaa38dc47bc08bb1f45dfe6cf4:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9a94015f209c90a9d6bf3352e302de8a9c44aa61-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9a94015f209c90a9d6bf3352e302de8a9c44aa61-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9a94015f209c90a9d6bf3352e302de8a9c44aa61-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9a94015f209c90a9d6bf3352e302de8a9c44aa61-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d99671aa966c288841b4261002babc0c7e944f73:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-fa7f4ee8d8361a21b2cf25b99f9cf3860c2c8bcb-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-fa7f4ee8d8361a21b2cf25b99f9cf3860c2c8bcb-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-fa7f4ee8d8361a21b2cf25b99f9cf3860c2c8bcb-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-fa7f4ee8d8361a21b2cf25b99f9cf3860c2c8bcb-x86.apk)

File: CMakeLists.txt [hunk 1]
```diff
@@ -105,6 +114,9 @@ if (ANDROID)
 
   include(CreateZip)
 
+  # Copy QCA plugins in the apprpriate crypto directory (under libs)
+  file(COPY "${OSGEO4A_LIB_DIR}/qca-qt5/crypto/" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/android-build/lib/${ANDROID_ABI}/crypto/")
```
Comment (nirvn) [inline CMakeLists.txt:118]:
```suggestion
  file(COPY "${OSGEO4A_LIB_DIR}/qca-qt5/crypto/" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/android-build/assets/share/crypto/")
```

**Comment by nirvn** [comment]
For those following the saga: qca requires plugins to be placed inside a crypto/ subdirectory. We can't have subdirectories in the APK's lib directory, so the alternative would be to move the Qt plugins directory to sit in share, and set the QApplication pluginPath to  that instead. We'd need to move the qgis providers and authentication methods there too.  Will try tomorrow.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 37bf44ae05a2bcdf679dae2e432360609de160e7:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9c45384a9b556dc75c2954ee6d28daed7640db7d-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9c45384a9b556dc75c2954ee6d28daed7640db7d-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9c45384a9b556dc75c2954ee6d28daed7640db7d-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-9c45384a9b556dc75c2954ee6d28daed7640db7d-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f80447a6d8de40eec9f882cdd40ce491cb59a2ec:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4cbb192c073db45304b19f80244c74d7a94d43e3-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4cbb192c073db45304b19f80244c74d7a94d43e3-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4cbb192c073db45304b19f80244c74d7a94d43e3-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-4cbb192c073db45304b19f80244c74d7a94d43e3-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0d0651d54405f41cea98649643c3f49a553d9077:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-103b36ead43b9620fd02c2c57849544ec071cac4-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-103b36ead43b9620fd02c2c57849544ec071cac4-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-103b36ead43b9620fd02c2c57849544ec071cac4-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-103b36ead43b9620fd02c2c57849544ec071cac4-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 04a46c3279eeb42b2d8774266943b0cae3e8b5f4:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c56bb38b72677bb5d7a9edabde89319c3a497ee8-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c56bb38b72677bb5d7a9edabde89319c3a497ee8-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c56bb38b72677bb5d7a9edabde89319c3a497ee8-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c56bb38b72677bb5d7a9edabde89319c3a497ee8-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 86ba8265a346451afcfe18a2a637d86c4ab7f85c:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-eeb6ffa8ca7818dbc520238127c7ed0ddcb6ac62-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-eeb6ffa8ca7818dbc520238127c7ed0ddcb6ac62-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-eeb6ffa8ca7818dbc520238127c7ed0ddcb6ac62-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-eeb6ffa8ca7818dbc520238127c7ed0ddcb6ac62-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7bf628e85cf32a97eafce40a3e017e6b880ea1cc:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-322c165dd8c22c5fd9c2bc7350a85da69c3b220b-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-322c165dd8c22c5fd9c2bc7350a85da69c3b220b-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-322c165dd8c22c5fd9c2bc7350a85da69c3b220b-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-322c165dd8c22c5fd9c2bc7350a85da69c3b220b-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 72ad547b5cfdf29b3e644438d5c67363b0fd84de:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3cfe0e01fb1d919d1e328351639417e511c78b45-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3cfe0e01fb1d919d1e328351639417e511c78b45-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3cfe0e01fb1d919d1e328351639417e511c78b45-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3cfe0e01fb1d919d1e328351639417e511c78b45-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1ecc1d6fef003c18c1ed1c412944e77a9120b34a:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-98782a80001eeeba4e7c82e80958794b83bf6fb8-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-98782a80001eeeba4e7c82e80958794b83bf6fb8-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-98782a80001eeeba4e7c82e80958794b83bf6fb8-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-98782a80001eeeba4e7c82e80958794b83bf6fb8-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 5299bd0f5ecf4ebd6961be61cbeb0f1179e538db:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3fdf39d31042cc90cb7d0c2a9fd061f03236a9be-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3fdf39d31042cc90cb7d0c2a9fd061f03236a9be-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3fdf39d31042cc90cb7d0c2a9fd061f03236a9be-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3fdf39d31042cc90cb7d0c2a9fd061f03236a9be-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 473fa7abbe9e722d31464552d65a371679624ff5:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8b315b2af80fd65704cba7b6f0c3e8a0f8052cc6-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8b315b2af80fd65704cba7b6f0c3e8a0f8052cc6-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8b315b2af80fd65704cba7b6f0c3e8a0f8052cc6-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-8b315b2af80fd65704cba7b6f0c3e8a0f8052cc6-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 51b85ee1716e032fdc7aa15552e975ab48d23d2b:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3b9d69aab395d30ea6dded7011f1e5ea2309f47a-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3b9d69aab395d30ea6dded7011f1e5ea2309f47a-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3b9d69aab395d30ea6dded7011f1e5ea2309f47a-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-3b9d69aab395d30ea6dded7011f1e5ea2309f47a-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d262b78991a474a0012cfe19c40d3624de974b9e:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-1778a73540db2d79f3cc789072d8a2eee2f03738-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-1778a73540db2d79f3cc789072d8a2eee2f03738-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-1778a73540db2d79f3cc789072d8a2eee2f03738-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-1778a73540db2d79f3cc789072d8a2eee2f03738-x86.apk)

File: android/src/ch/opengis/qfield/QFieldActivity.java [hunk 2]
```diff
@@ -98,6 +98,7 @@ private void prepareQtActivity() {
         new File(qFieldDir + "basemaps/").mkdir();
         new File(qFieldDir + "fonts/").mkdir();
         new File(qFieldDir + "proj/").mkdir();
+        new File(qFieldDir + "authentications/").mkdir();
```
Comment (m-kuhn) [inline android/src/ch/opengis/qfield/QFieldActivity.java:101]:
```suggestion
        new File(qFieldDir + "auth/").mkdir();
```

what about making this shorter?

Comment (nirvn) [reply android/src/ch/opengis/qfield/QFieldActivity.java:101]: No strong feeling, we can go with `auth`.

Comment (nirvn) [reply android/src/ch/opengis/qfield/QFieldActivity.java]: @m-kuhn , done.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 18a41b793df18efb23332f512f905724da54fbf9:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-21d87e43d903475a746b138a49cd3e82a0ba8a5a-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-21d87e43d903475a746b138a49cd3e82a0ba8a5a-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-21d87e43d903475a746b138a49cd3e82a0ba8a5a-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-21d87e43d903475a746b138a49cd3e82a0ba8a5a-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9bdfcb42ee5c4eba569d8ed6b0ee4627bfaaad79:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ca1af26c17eb0850fc6650eb3bd7f8686c159aef-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ca1af26c17eb0850fc6650eb3bd7f8686c159aef-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ca1af26c17eb0850fc6650eb3bd7f8686c159aef-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ca1af26c17eb0850fc6650eb3bd7f8686c159aef-x86.apk)

**Comment by m-kuhn** [comment]
Something to keep in mind, loading dynamic libs is not supported by ios, so we'll have to check how to statically build these modules into qgis (like the providers)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 85a8dd92c50ac9b7744d249373fbf4583c448072:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-590fdc02c4422ce32b7a352c2583981f27184d27-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-590fdc02c4422ce32b7a352c2583981f27184d27-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-590fdc02c4422ce32b7a352c2583981f27184d27-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-590fdc02c4422ce32b7a352c2583981f27184d27-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for c6339bfd418718d2e90ccba39e150e3d26c0c858:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19d63ba465a68bd80e3a540c3673be7f82134e65-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19d63ba465a68bd80e3a540c3673be7f82134e65-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19d63ba465a68bd80e3a540c3673be7f82134e65-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-19d63ba465a68bd80e3a540c3673be7f82134e65-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7a95d8834966ecadd2ef55b72d74bf0ba68d91ab:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c692e0103624198fcfc82de6d1afc2f3e03e26fa-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c692e0103624198fcfc82de6d1afc2f3e03e26fa-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c692e0103624198fcfc82de6d1afc2f3e03e26fa-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-c692e0103624198fcfc82de6d1afc2f3e03e26fa-x86.apk)

**Comment by nirvn** [comment]
@m-kuhn , alright, the APK is back to 60mb, we're now manually stripping /share/plugins libraries.

**Comment by m-kuhn** [comment]
Good good. Now just those win builds

**Comment by nirvn** [comment]
@m-kuhn , the windows build failure we are facing is this (https://stackoverflow.com/questions/13416418/define-nominmax-using-stdmin-max/13420838) -- no idea how to properly fix it though.

File: CMakeLists.txt [hunk 3]
```diff
@@ -172,6 +172,7 @@ set (ENABLE_TESTS TRUE CACHE BOOL "Build unit tests?")
 if(MSVC)
   find_package(Qt5 COMPONENTS Charts REQUIRED) # vcpkg doesn't include QtCharts.dll as dep of the qml module otherwise
   add_definitions(-D_USE_MATH_DEFINES)
+  add_definitions(-DNOMINMAX)
```
Comment (nirvn) [inline CMakeLists.txt:175]: @m-kuhn , this might to it, crossing fingers.

**Comment by nirvn** [comment]
@m-kuhn , windows is green :tada: let's merge this and never look back! :)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for b7802b5d0b0d1346427e7b0de6c20715642411b3:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ba8e5841830f156b04bdaaad0b3508e153ae48d1-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ba8e5841830f156b04bdaaad0b3508e153ae48d1-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ba8e5841830f156b04bdaaad0b3508e153ae48d1-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1907-ba8e5841830f156b04bdaaad0b3508e153ae48d1-x86.apk)

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**